CHANGELOG.next.asciidoc

```diff
@@ -118,6 +118,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `socket_summary` metricset to system defaults, removing experimental tag and supporting Windows {pull}9709[9709]
 - Add docker `event` metricset. {pull}9856[9856]
 - Add 'performance' metricset to x-pack mssql module {pull}9826[9826]
+- Add more meaningful metrics to 'performance' Metricset on 'MSSQL' module {pull}10011[10011]
```

Comment:
As soon as we have all PR's related to MSSQL in, we can probably merge the changelog entries a bit for the release (not now).